packages/eas-cli/src/commandUtils/__tests__/EasCommand-test.ts

```diff
@@ -96,5 +101,85 @@ describe(EasCommand.name, () => {
 
       expect(analytics.flushAsync).toHaveBeenCalled();
     });
+
+    describe('catch', () => {
+      it('logs the message', async () => {
+        const TestEasCommand = createTestEasCommand();
+        const logErrorSpy = jest.spyOn(Log, 'error');
+        const logDebugSpy = jest.spyOn(Log, 'debug');
+        const runAsyncMock = jest.spyOn(TestEasCommand.prototype as any, 'runAsync');
+        const error = new Error('Unexpected, internal error message');
+        runAsyncMock.mockImplementation(() => {
+          throw error;
+        });
+        try {
+          await TestEasCommand.run();
+        } catch {}
+
+        expect(logErrorSpy).toBeCalledWith('Unexpected, internal error message');
+        expect(logDebugSpy).toBeCalledWith(error);
+      });
```

Comment:
Thanks for adding tests 👍

Reply:
`Record<string, typeof Error>` gives the same error as `{ [key: string]: any }` sadly

Reply:
I think you commented under wrong line but try what I suggested there `Record<string, new () => Error>`.